orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/DeploymentDetailsAware.groovy

```diff
@@ -133,7 +133,7 @@ trait DeploymentDetailsAware {
     if (execution instanceof Pipeline) {
       if (execution.trigger.parentExecution instanceof Pipeline) {
         return execution.trigger.parentExecution
-      } else if (execution.trigger.parentExecution) {
+      } else if (execution.trigger.parentExecution?.isPipeline) {
```

Comment:
Pull this up to `execution.trigger` and take out the `JsonIgnoreProperties`, and I think we'd be good.

Should be easy enough to test this.